### PR TITLE
added optional wind conditions

### PR DESCRIPTION
**Pull request by morozgrafix**
Added a way to optionally display wind speed and direction

It looks like this:
image

### CODE REVIEW

File: MMM-forecast-io.js [hunk 1]
```diff
@@ -185,6 +186,57 @@ Module.register("MMM-forecast-io", {
...
+      var cardinalDirection = this.translate('WSW');
+    } else if (currentWeather.windBearing > 258.75 && currentWeather.windBearing <= 281.25) {
+      var cardinalDirection = this.translate('W');
+    } else if (currentWeather.windBearing > 281.25 && currentWeather.windBearing <= 303.75) {
+      var cardinalDirection = this.translate('WNW');
+    } else if (currentWeater.windBearing > 303.75 && currentWeather.windBearing <= 326.25) {
+      var cardinalDirection = this.translate('NW');
+    } else {
+      var cardinalDirection = this.translate('NNW');
+    }
```
Comment (dmcinnes) [inline MMM-forecast-io.js]: I think all these if/else statements could be avoided. How about divide the wind bearing (plus 11.25) by 22.5 to get an index on an Array with the `cardinalDirection`? `['N', 'NNE', 'NE', ...`

File: README.md [hunk 2]
```diff
@@ -105,7 +105,14 @@ modules: [
...
       </td>
-    </tr><tr>
+    </tr>
+    <tr>
+      <td><code>showWind</code></td>
+      <td>Toggles display of wind conditions<br>
+        <br><b>Default value:</b>  <code>true</code>
+      </td>
+    </tr>
+    <tr>
```
Comment (dmcinnes) [inline README.md:115]: Thanks for this!

**Review by dmcinnes — CHANGES_REQUESTED** [review]
Looks great! Just a suggestion on how to avoid the if/elses.

File: MMM-forecast-io.js (hunk 1 above)
Comment (morozgrafix) [reply MMM-forecast-io.js]: Good point. I will change that logic.

**Comment by morozgrafix** [comment]
Cardinal Directions logic is changed and hopefully simplified a bit. Thanks!

**Comment by dmcinnes** [comment]
Thanks! Sorry lagging on reviewing.

**Pull request merged**